client/src/components/dataset/DatasetTable.jsx

```diff
@@ -114,136 +54,118 @@ export default class DatasetTable extends Component {
   }
 
   getCellClassName(columnTitle) {
-    if (this.state.activeSidebar &&
-      this.state.activeSidebar.columnTitle === columnTitle) {
+    const { sidebarProps } = this.state;
+    if (sidebarProps != null && sidebarProps.columnTitle === columnTitle) {
```

Comment:
This works for "Filter" sidebar, but not for the change-data-type sidebar as that doesn't have the `columnTitle` property. For this to work correctly we need to also check if `sidebarProps.column.title === columnTitle`